### DISCUSSION TITLE
how to remove white divider line between Drawer and Screen content?

### DISCUSSION THREAD

**Discussion by adilusman51** [react-navigation/react-navigation#10787] [Q&A]
Hi, I am using `@react-navigation/drawer: ^6.4.3` in my project and when I switch theme to Dark Mode, a white line is shown between Drawer and Screen content, as can be seen in the image below

<img width="555" alt="Screen Shot 2022-08-25 at 1 24 37 AM" src="https://user-images.githubusercontent.com/26340587/186519204-542b0e65-ac24-4188-bc5d-00c509a5eff5.png">

This line in not visible in Light theme, may be due to the white color of the line.

Kindly, guide me how to style this white line to match the current theme of App (Light or Dark)

**Comment by Ryrilya** [comment]
@adilusman51 I know it’s already too late, but I haven’t seen any answer to this issue so this might help others.
I solved it by putting the properties `borderLeftWidth: 0` and/or `borderRightWidth: 0` to `screenOptions.drawerStyle` of the drawer `Navigator`.

e.g:
```tsx
const SCREEN_OPTIONS: DrawerNavigationOptions = {
 …,
 drawerStyle: {
  …,
  borderLeftWidth: 0,
  borderRightWidth: 0,
 },
};

<MainDrawer.Navigator
        ...
 screenOptions={SCREEN_OPTIONS}
>
   ...
</MainDrawer.Navigator>
```
